cloudflare_cache/cloudflare_cache.php

```diff
@@ -0,0 +1,36 @@
+<?php
+// TODO: this header should not be required...
+header('Content-Type: text/plain; charset=UTF-8');
+
+// Only purge Cloudflare cache when the live environment's cache is cleared.
+if ($_ENV['PANTHEON_ENVIRONMENT'] != 'live') {
+  die();
+}
+
+// Retrieve Cloudflare config data
+$config_file = $_SERVER['HOME'] . '/files/private/cloudflare_cache.json';
+$config = json_decode(file_get_contents($_SERVER['HOME'] . '/files/private/cloudflare_cache.json'), 1);
+if ($config == FALSE) {
+  die('files/private/cloudflare_cache.json found. Aborting!');
```

Comment:
Not found. Break out error checking per slack example; we had a debugging problem with another sample where the error message was saying "file not found", but what it meant was that the file was found, but the json was invalid.